src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -1,19 +1,42 @@
 package redis.clients.jedis.tests;
 
+import java.io.IOException;
```

Comment:
Can you please revert the import changes?. This file doesn't need to be modified.